crates/uv-git/src/source.rs

```diff
@@ -82,7 +82,9 @@ impl GitSource {
 
         // Authenticate the URL, if necessary.
         let remote = if let Some(credentials) = GIT_STORE.get(&canonical) {
-            Cow::Owned(credentials.apply(self.git.repository().clone()))
+            Cow::Owned(LogSafeUrl::from(
+                credentials.apply(Url::from(self.git.repository().clone())),
```

Comment:
`credentials.apply` should operate on a redacting URL by default

Reply:
The reason is that I've intentionally handled the `uv_auth` part of the work in a separate PR (#13576). `Credentials` does operate on the new type there.